nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults

```diff
@@ -39,6 +39,9 @@ nuxeo.vcs.fulltext.analyzer.language=english
 nuxeo.vcs.optimizations.acl.enabled=true
 nuxeo.vcs.optimizations.path.enabled=true
 
+# Enable reliable change tokens in VCS/DBS
+nuxeo.changetoken.enabled=true
+
```

Comment:
It's not really an admin param, it's a variable to change the default value in the default-repository-config.xml template. But otherwise contributing a full extension point for the repository is the way to go if more complex changes are needed.